west.yml

```diff
@@ -52,7 +52,7 @@ manifest:
     # https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/zephyr/guides/modules.html
     - name: zephyr
       repo-path: sdk-zephyr
-      revision: v2.7.99-ncs1-rc1
+      revision: pull/726/head
```

Comment:
Please udpate SHA: `d5a653348eeab5a3bd6305e4f1cd2d6dcf56774a` (note that it contains some extra commits to avoid double update of the manifest).